### DISCUSSION TITLE
Add existing functionality to hidden files

### DISCUSSION THREAD

**Discussion by qaeu** [ente-io/ente#3834] [Enhancements]
There's a few existing features that are not available to hidden files but I feel should be:
* Searching. No search bar at all when browsing hidden items currently
* De-duplication, and by extension ignoring duplicate uploads to hidden albums
* Moving unhidden files to specific hidden albums
* Moving hidden files to other albums (on android we seem to be able to move them to other hidden albums)
* Machine Learning / people finding

**Comment by huntfx** [comment]
To add to this, support the "All" and "Favourite" categories would be great too, since currently it's just the equivalent of "Uncategorised" (which is empty for me as I only hide full albums).

**Comment by Gadetron** [comment]
to add to this, add the ability to upload photos from watched folders on pc to hidden albums, as of now it just creates a new album of the same name instead of uploading to the hidden one i want
